docs/dev_docs/HowToTutorial.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Making Tutorials
+category: developer documentation
+tags: tutorial, contribute
+layout: default_toc
+---
+
+<script type="text/javascript" async
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML">
+</script>
+<link rel="stylesheet" href="css/main.css">
+
+This project was supported by grants from the National Institute of Biomedical Imaging and Bioengineering (U24EB029012) from the National Institutes of Health.
+
+
+Authors:  
+Jess Tate 
+
+### Contents
+
+*  [Overview](#overview)
+	-  [Software Requirements](#software-requirements)
+*  [Files Needed for a New Tutorial](#files-needed-for-a-new-tutorial)
+	-  [Overview of Files Needed for a Tutorial](#overview-of-files-needed-for-a-tutorial)
+	-  [Markdown File](#markdown-file)
+	-  [Added Figures](#added-figures)
+       -  [Additional Files](#additional-files)
+       -  [Linking to New Tutorial](#linking-to-new-tutorial)
+*  [Testing Documentation](#testing-documentation)
+       - [Testing Locally](#testing-locally)
+            + [Installing Jekyll](#installing-jekyll)
+            + [Building Documentation](#building-documentation)
+       - [Testing on a Fork](#testing-on-a-fork)
+* [Adding Content](#adding-content)
+       - [Figures](#figures)
+       - [Math](#math)
+       - [Citations](#citations)
+       - [Snippets](#snippets)
+       - [Links](#links)
+       -[Referencing Sphynx](#referencing-sphynx)
+* [Content Guide](#content-guide)
+* [Supplemental Materials](#supplemental-materials)
+       - [Example Scripts](#example-scripts)
+       - [Movies](#movies)
+       - [Datasets](#datasets)
+
+### Overview
+
+**This tutorial demonstrates how to use markdown to create new tutorials for UcertainSCI.  It will walk through all the files needed and the basic structure needed expected for tutorials.  Knowledge of Markdown, Github, Github pages, and Python will be useful. If you have questions, [please ask](https://github.com/SCIInstitute/UncertainSCI/discussions).**
```

Comment:
```suggestion
**This tutorial demonstrates how to use markdown to create new tutorials for UncertainSCI.  It will walk through all the files needed and the basic structure needed expected for tutorials.  Knowledge of Markdown, Github, Github pages, and Python will be useful. If you have questions, [please ask](https://github.com/SCIInstitute/UncertainSCI/discussions).**
```